CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+### 2.1.0 (unreleased)
+
+New features:
+ * `WhitelistCrowdsale`: a crowdsale where only whitelisted accounts (`WhitelistedRole`) can purchase tokens. This attribute is given by a series of whitelisters (`WhitelisterRole`), who can also remove them. Similar to the pre-2.0 `WhitelistedCrowdsale`.
+ * `RefundablePostDeliveryCrowdsale`: replacement for the (deprecated) `RefundableCrowdsale`: tokens are only granted once the crowdsale ends (if it meets its goal).
+ * `ERC20`: `transferFrom` and `_burnFrom ` now emit `Approval` events, improving dApp support.
```

Comment:
I found many of these descriptions to be too vague. What does "improving dApp support" mean?

I'd prefer something like this. What do you think?
```suggestion
 * `ERC20`: `transferFrom` and `_burnFrom ` now emit `Approval` events, to represent the token's state comprehensively through events.
```

Reply:
I do think my original description was too vague, but I'd like for us to explain _why_ 'representing the token's state comprehensively through events' is important.

Reply:
I agree. Do you think the changelog is the right place for that? Perhaps we should have a link to "read more"?